e2e/benchmarks/local-benchmark/loader.js

```diff
@@ -60,7 +60,6 @@ async function loadTFJS(localBuild) {
     'https://cdn.jsdelivr.net/npm/@tensorflow-models/speech-commands',
     'https://cdn.jsdelivr.net/npm/@tensorflow-models/posenet@2',
     'https://cdn.jsdelivr.net/npm/@tensorflow-models/body-pix@2',
-    'https://cdn.jsdelivr.net/npm/@tensorflow-models/pose-detection',
```

Comment:
It seems that this package is not used anymore in benchmarks. However, the latest update of `pose-detection` causes webgpu backend uploaded twice (one is tfjs-backend-webgpu itself in L 50 and the other one is pose-detection's tfjs-backend-webgpu), which throws warnings like below:
```
The kernel '_FusedMatMul' for backend 'webgpu' is already registered
warn @ log.ts:22
registerKernel @ kernel_registry.ts:135
(anonymous) @ pose-detection:17
(anonymous) @ pose-detection:17
(anonymous) @ pose-detection:17
log.ts:22 The kernel 'Abs' for backend 'webgpu' is already registered
warn @ log.ts:22
registerKernel @ kernel_registry.ts:135
(anonymous) @ pose-detection:17
(anonymous) @ pose-detection:17
(anonymous) @ pose-detection:17
log.ts:22 The kernel 'Acos' for backend 'webgpu' is already registered
warn @ log.ts:22

... ...
``` 
@axinging It will be great if you can do further investigation that why only webgpu has those warnings. My understanding is webgl should have the similar issue.

@mattsoulanille @chunnienc If you know the possible reasons, please let us know. Thanks.

Reply:
Maybe because [commit1](https://github.com/tensorflow/tfjs-models/commit/22813322d49170b7c024635f7dc962f38ee9a41f#diff-1a6370034503c27e8ed6f5bed70d97fc979eab7493993708d9469086e1436504) or [commit 2](https://github.com/tensorflow/tfjs-models/commit/04b1051c881e209cc8e1a5b0079b75e14ac048a1#diff-1a6370034503c27e8ed6f5bed70d97fc979eab7493993708d9469086e1436504) added webgpu into the peer dependencies? I can see the entire webgpu backend is bundled in the js file from jsdelivr while other backends are not.

It might be possible to mark webgpu backend as external here https://github.com/tensorflow/tfjs-models/blob/master/pose-detection/rollup.config.js. But since I didn't see any webgpu backend imports from the model src, it will be better to figure out why rollup packs it into the bundle.

Reply:
Thanks @chunnienc, this can be fixed by external: https://github.com/tensorflow/tfjs-models/pull/1140/files